adapter/ipListChecker/BUILD

```diff
@@ -12,10 +12,12 @@ go_library(
     deps = [
         "@com_github_golang_glog//:go_default_library",
         "@in_gopkg_yaml_v2//:go_default_library",
+        "//pkg/adapter:go_default_library",
```

Comment:
Nit: in general, it might be nice if we decided on a convention for build file formats. I suggest using [buildifier](https://github.com/bazelbuild/buildifier). this would end up having these deps be sorted.